lib/browser/deps/livereload-client.js

```diff
@@ -35,7 +35,7 @@ function loadScript(src, lrload) {
 
   function tryLoad() {
     var script = document.createElement('script')
-    script.setAttribute('src', src);
+    script.setAttribute('src', src + '?bust=' + (new Date()));
```

Comment:
How about if `src` contains already query parameters? 

`has_params ? '&' : '?'` must be added